src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1124,6 +1216,38 @@ private Task TransmitAsync(JsonRpcMessage message, CancellationToken cancellatio
             return this.MessageHandler.WriteAsync(json, cancellationToken);
         }
 
+        internal struct MethodTarget : IEquatable<MethodTarget>
```

Comment:
`MethodTarget` makes sense given its content, but we already have a `TargetMethod` type, so that's potentially very confusing. Any suggestions for renaming either type?